RetailCoder.VBE/UI/DockableToolwindowPresenter.cs

```diff
@@ -100,6 +100,23 @@ private void EnsureMinimumWindowSize(IWindow window)
         public virtual void Show() => _window.IsVisible = true;
         public virtual void Hide() => _window.IsVisible = false;
 
+
+        private bool _isDisposed;
+        public void Dispose()
```

Comment:
I wonder why we are not also implementing `protected virtual void Dispose(bool disposing)` as documented in the [Dispose pattern](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/dispose-pattern).

Reply:
Will read and then act accordingly.

Reply:
Seems appropriate at first glance, given this class is inherited, but all implementations merely pass the COM object right here... I doubt it would make a difference.

Reply:
I'll implement it anyway. Atm, I am not too sure whether we have some wrappers caching wrappers they newed up themselves based on values returned from the target. Those would be a candidate for overriding the `Dispose` method.